precompiles/contracts.go

```diff
@@ -65,46 +66,46 @@ func Add(utype byte, lhsHash []byte, rhsHash []byte, tp *TxParams) ([]byte, uint
 	}
 
 	if shouldPrintPrecompileInfo(tp) {
-		logger.Info("starting new precompiled contract function ", functionName)
+		logger.Info("Starting new precompiled contract function: " + functionName)
```

Comment:
```suggestion
		logger.Info("Starting new precompiled contract function:", functionName)
```

shouldn't it be like that here and in every other location?

Reply:
I now use go-ethereum's `log` module, which defines the `logger.Info`, `Trace`, etc. functions as functions that receive the message as the first arguments, and every next two arguments are a key and a value that gets attached to the log.
Therefore the message must be a single argument, not two. Alternatively, we may do something like `logger.Info("Starting function", "function", functionName)`.

e.g. in the picture below there's a "module=fheos" key-value from the arguments "module", "fheos". (I made this automatic on every fheos log by supplying it to the `New()` logger.
![image](https://github.com/FhenixProtocol/fheos/assets/98821241/b01112c2-7149-408d-aaee-3fb6d6b08886)